packages/tools/pluggable-widgets-tools/src/typings-generator/generatePreviewTypes.ts

```diff
@@ -4,8 +4,10 @@ import { capitalizeFirstLetter, extractProperties } from "./helpers";
 export function generatePreviewTypes(widgetName: string, properties: Property[]): string[] {
     const results = Array.of<string>();
     results.push(`export interface ${widgetName}PreviewProps {
-    class: string;
-    style: string;
+    className: string;
+    readOnly: boolean;
+    style?: string;
+    styleObject?: CSSProperties;
```

Comment:
Question: what's the difference between `style` and `styleObject`? Same content but different types (string vs object)?

Reply:
styleObject apparently is a parsed object introduce in Mendix 8.3 (Which i was not aware). I've sync with Fabian about those props.